src/24-hour-rule.ts

```diff
@@ -178,13 +178,16 @@ export function setUp24HourRule(probot: Probot) {
       probot.log(`Found ${prs.length} prs for repo: ${repo.owner.login}/${repo.name}`);
 
       for (const pr of prs) {
-        await applyLabelToPR(
-          github,
-          pr as any,
-          repo.owner.login,
-          repo.name,
-          shouldPRHaveLabel(pr as any),
-        );
+        const shouldLabel = shouldPRHaveLabel(pr as any);
+
+        // We also need to ensure that API review labels are updated when the requisite
+        // waiting period expires.
+        if (!shouldLabel) {
```

Comment:
This should be some kind of `if (!shouldLabel && currentlyHasLabel)` check right? Otherwise we're running this on every PR every 5 minutes

Reply:
oh good point, yeah will do!